packages/celer-web-app/src/ui/frames/AppFrame.tsx

```diff
@@ -51,7 +54,205 @@ export const AppFrame: React.FC<EmptyObject> = ()=>{
 			errorCount++;
 		}
 	});
+	if(useNew)
+	{
+		return (
+			<div className={styles.appFrame}>
+
+				<DocFrame docLines={docLines} />
+
+				<div className={styles.statusBarFrame}
+					style={{ zIndex:showMenu?99999:-1}}
+					onClick={()=>{
+						setShowMenu(false);
+						setContextMenuRef(undefined);
+					}} >
+					<div className={styles.menuOverlayFrame}style={showMenu?{ height: "auto" } : undefined}>
+
+						{showMenu && <>
+							{contextMenuRef === splitSettingsMenuItemRef && <div className={styles.submenu} style={{
+								bottom: `calc( 100vh - ${contextMenuRef.current?.getBoundingClientRect().bottom || 0}px )`,
+							}}>
+								<MenuItemWithValue value={getSplitSettingText(setting.splitSettings[SplitType.Shrine])} action={function (): void {
+									setSetting(
+										produce(setting, (draft) => {
```

Comment:
You don't need to nest `produce` with `immer`. see https://immerjs.github.io/immer/update-patterns/#nested-data-structures